lib/features/registration/pages/us_signup_page.dart

```diff
@@ -80,13 +81,11 @@ class _UsSignUpPageState extends State<UsSignUpPage> {
 
         return FamilyScaffold(
           appBar: GenerosityAppBar(
-            leading: context.canPop()
-                ? GenerosityBackButton(
+            leading: GenerosityBackButton(
                     onPressed: () {
-                      context.pop();
+                      logout(context, fromLogoutBtn: true);
```

Comment:
Logging out will cause a redirect in the router?

Reply:
yes